census_rm_ops_ui/templates/view_case_details.html

```diff
@@ -0,0 +1,21 @@
+{% extends "layouts/base.html" %}
+
+{% block main %}
+    <main id="main-content" class="page__main "></main>
+    <p class="field u-mb-l">
+        <a href={{ url_for('postcode_search_bp.postcode_results', postcode=case_details.postcode) }}>Back to search results</a>
+            </p>
+    <h1>Case reference {{ case_details.caseRef }}</h1>
+
+    {%  for key, value in case_details.items() %}
+        <p><b>{{ key }}</b> : {{ value }}</p>
+    {% endfor %}
+    <h1>Event history</h1>
+    {% for event in case_events %}
+        {%  for key, value in event.items() %}
+            <p>{{ key }} : {{ value }}</p>
```

Comment:
Could make this key bold the same as above. Normally leave a space after the colon not before too.